JetNews/app/src/main/java/com/example/jetnews/ui/home/PostCards.kt

```diff
@@ -42,115 +42,104 @@ import com.example.jetnews.model.Post
 import com.example.jetnews.ui.JetnewsStatus
 import com.example.jetnews.ui.Screen
 import com.example.jetnews.ui.ThemedPreview
-import com.example.jetnews.ui.VectorImage
 import com.example.jetnews.ui.darkThemeColors
 import com.example.jetnews.ui.navigateTo
 
 @Composable
 fun AuthorAndReadTime(post: Post) {
     Row {
-
-        val textStyle = MaterialTheme.typography().body2
-        ProvideEmphasis(emphasis = EmphasisLevels().medium) {
+        val textStyle = MaterialTheme.typography.body2
+        ProvideEmphasis(EmphasisAmbient.current.medium) {
             Text(text = post.metadata.author.name, style = textStyle)
-            Text(
-                text = " - ${post.metadata.readTimeMinutes} min read",
-                style = textStyle
-            )
+            Text(text = " - ${post.metadata.readTimeMinutes} min read", style = textStyle)
         }
     }
 }
 
 @Composable
-fun PostImage(modifier: Modifier = Modifier.None, post: Post) {
+fun PostImage(post: Post, modifier: Modifier = Modifier) {
```

Comment:
Again, shouldn't the default be `Modifier.None`?

Reply:
inquiring - they appear to be exact aliases so figuring out what was intended.